File: CS-Examples/11_Formatting/GetStyleSetStyle.cs
using Spire.Xls;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Xls.Core.Spreadsheet;

namespace GetStyleSetStyle
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create a workbook
            Workbook workbook = new Workbook();

            //Load a excel document
            workbook.LoadFromFile(@"..\..\..\..\..\..\Data\templateAz.xlsx");
            
            //Get the first sheet
            Worksheet sheet = workbook.Worksheets[0];

            //Get "B4" cell
            CellRange range = sheet.Range["B4"];       
            //Get the style of cell
            CellStyle style = range.Style;
            style.Font.FontName = "Calibri";
            style.Font.IsBold = true;
            style.Font.Size = 15;
            style.Font.Color = Color.CornflowerBlue;

            range.Style = style;

            String result = "UseGetStyleSetStyle_result.xlsx";

            //Save to file
            workbook.SaveToFile(result, ExcelVersion.Version2010);
           
            //View the document
           FileViewer(result);
        }

        private void FileViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            Close();
        }
    }
}

File: VB-Examples/11_Formatting/GetStyleSetStyle.vb
Imports Spire.Xls
Imports System.ComponentModel
Imports System.Text
Imports Spire.Xls.Core.Spreadsheet

Namespace GetStyleSetStyle
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
            ' Create a new Workbook object
            Dim workbook As New Workbook()

            ' Load the template file from the specified path
            workbook.LoadFromFile("..\..\..\..\..\..\Data\templateAz.xlsx")

            ' Get the first worksheet in the workbook
            Dim sheet As Worksheet = workbook.Worksheets(0)

            ' Specify the range as cell B4 in the worksheet
            Dim range As CellRange = sheet.Range("B4")

            ' Get the style of the specified range
            Dim style As CellStyle = range.Style

            ' Set the font properties of the style
            style.Font.FontName = "Calibri"
            style.Font.IsBold = True
            style.Font.Size = 15
            style.Font.Color = Color.CornflowerBlue

            ' Apply the modified style to the range
            range.Style = style

            ' Save the modified workbook to a new file named "UseGetStyleSetStyle_result.xlsx" in Excel 2010 format
            Dim result As String = "UseGetStyleSetStyle_result.xlsx"
            workbook.SaveToFile(result, ExcelVersion.Version2010)

            ' Release the resources used by the workbook
            workbook.Dispose()

            'View the document
            FileViewer(result)
		End Sub

		Private Sub FileViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

		Private Sub btnClose_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnClose.Click
			Close()
		End Sub
	End Class
End Namespace
